hexrd/unitcell.py

```diff
@@ -1246,6 +1209,50 @@ def color_orientations(self,
 
         self.stiffness = C
 
+    def is_editable(self, lp_name):
+        """
+        @author Saransh Singh, Lawrence Livermore National Lab
+        @date 03/17/2021 SS 1.0 original
+        @details check if a certain field in the lattice parameter 
+        is editable. this depends on the space group number or the
+        lattice class
+        """
+        res = False
+        _lpnamelist = list(_lpname)
+        index = _lpnamelist.index(lp_name)
+        editable_fields = _rqpDict[self.latticeType][0]
+        if(index in editable_fields):
+            res = True
+        return res
+
+    def convert_lp_to_valunits(self, lp):
+        """
+        added 03/17/2021 SS
+        """
+        lp_valunit = []
+        for i in range(6):
+            if(i < 3):
+                lp_valunit.append(
+                    valWUnit('lp', 'length',  lp[i], 'nm'))
+
+            else:
+                lp_valunit.append(
+                    valWUnit('lp', 'angle',  lp[i], 'degrees'))
+
+        return lp_valunit
+
+    def fill_correct_lp_vals(self, lp, val, lp_name):
+        """
+        added 03/17/2021 SS
+        """
+        index = list(_lpname).index(lp_name)
```

Comment:
Where is this `_lpname` variable defined?

Reply:
Same as above.